src/EA.Weee.RequestHandlers.Tests.Unit/AatfReturn/SubmitReturnHandlerTests.cs

```diff
@@ -114,7 +105,7 @@ public async Task HandleAsync_GivenReturn_StatusShouldBeUpdated()
 
             await handler.HandleAsync(message);
 
-            A.CallTo(() => @return.UpdateSubmitted(userId.ToString())).MustHaveHappened(Repeated.Exactly.Once)
+            A.CallTo(() => @return.UpdateSubmitted(userId.ToString(), false)).MustHaveHappened(Repeated.Exactly.Once)
```

Comment:
Can we change the test so that it is a [Theory] test and passes true & false to the test (i.e. for both nil return values) and then check that the UpdateSubmitted is called correctly with the value

Reply:
Added [Theory] test